rllib/evaluation/env_runner_v2.py

```diff
@@ -392,16 +394,21 @@ def _get_rollout_metrics(self, episode: EpisodeV2) -> List[RolloutMetrics]:
             for m in atari_metrics:
                 m._replace(custom_metrics=episode.custom_metrics)
             return atari_metrics
+        # Create connector metrics
+        connector_metrics = {}
+        for policy_id, policy in self._worker.policy_map.items():
+            connector_metrics[policy_id] = policy.get_connector_throughput_metrics()
```

Comment:
this could be pretty costly. any chance we can only update the policies that are used for these episodes?
at the very least, let's only update the cached poclies.

Reply:
Changed, thanks! 👍